docs/README.md

```diff
@@ -46,6 +47,43 @@ The `trustpilot` app also enables Truspilot's widgets to be added to your store
 | ------------------- | ------------------- | --------------------------------------- |
 | `shop-review-badge` | `trustpilot-widget` | Adds a Trustpilot widget in your store. |
 
+
+## Block without badge
+
+To use widget without `shop-review-badge` to landings for example you can use `trustpilot-widget`, you need add to your manifest this 
```

Comment:
```suggestion
To use widget without `shop-review-badge` to landings, for example, you can use `trustpilot-widget`. Therefore, you need to add the following in the manifest.json file.
```
Hugo, just to clarify, what do you mean by "landings"? Is there any word could we substitute for? Thank you!

Reply:
I think he meant landing pages

Reply:
fixed